app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+from typing import List
```

Comment:
1. There is a convention for order of imports that imports from build in libraries must come first. Also you good practice to separate different groups of imports by blank lines
2. You actually can you build-in `list` object for type annotations as of python 3.9